templates/payments/workshop_list.html

```diff
@@ -2,6 +2,14 @@
 
 {% block title %}QIIME - Workshops{% endblock %}
 {% block header %}Workshops{% endblock %}
+{% block extrahead %}
+<!--  Legal requirement for EU viewable websites https://www.cookielaw.org/the-cookie-law/ -->
```

Comment:
I [was looking into it a bit](https://www.cookielaw.org/faq#Whatdoesstrictlynecessarymean) and saw that for our sales portion, it isn't required. The site does use Google Analytics though, which isn't exempt.